pkg/kubectl/scale.go

```diff
@@ -199,51 +198,23 @@ func (scaler *ReplicationControllerScaler) Scale(namespace, name string, newSize
 		// Make it try only once, immediately
 		retry = &RetryParams{Interval: time.Millisecond, Timeout: time.Millisecond}
 	}
-	var updatedResourceVersion string
-	cond := ScaleCondition(scaler, preconditions, namespace, name, newSize, &updatedResourceVersion)
-	if err := wait.PollImmediate(retry.Interval, retry.Timeout, cond); err != nil {
+	cond := ScaleCondition(scaler, preconditions, namespace, name, newSize, nil)
+	if err := wait.Poll(retry.Interval, retry.Timeout, cond); err != nil {
```

Comment:
Why not PollImmediate as it was?

Reply:
Good catch.. It came as part of copy-pasting Scale() for replicaset. Changed it here and at other places too.